viewing_party/party.py

```diff
@@ -0,0 +1,145 @@
+# WAVE 1
+def create_movie(title, genre, rating):
+    if bool(title) and bool(genre) and bool(rating):
+        new_movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return new_movie
+    else:
+        return None
```

Comment:
To check for the truthiness of a variable in an expression we can use its name directly instead of using bool.

```suggestion
def create_movie(title, genre, rating):
    if title and genre and rating:
        new_movie = {
            "title": title,
            "genre": genre,
            "rating": rating
        }
        return new_movie
    else:
        return None
```

We can further improve on this logic by reversing the conditional and check for a 'failed' case early, allowing the 'main' logic to be unindented and have more of a focus in the function body. This is known as a guard clause and is a common pattern used in other languages like c, go, etc. 

```suggestion
def create_movie(title, genre, rating):
    if not title or not genre or not rating:
        return None
        
    new_movie = {
            "title": title,
            "genre": genre,
            "rating": rating
        }
    return new_movie
```
__